src/ui/Combobox/ComboboxValues.ts

```diff
@@ -54,13 +54,26 @@ export class ComboboxValues implements IComboboxValues {
         { id: `${this.combobox.id}-value-${index}`, className: 'coveo-combobox-value', role: 'option', tabindex: 0 },
         value.element
       ).el;
+      this.highlightCurrentQueryInSearchResults(value.element);
       value.element = elementWrapper;
       fragment.appendChild(value.element);
     });
 
     this.element.appendChild(fragment);
   }
 
+  private highlightCurrentQueryInSearchResults(searchResult: HTMLElement) {
+    if (this.combobox.options.highlightValueClassName) {
+      const regex = new RegExp('(' + this.combobox.element.querySelector('input').value + ')', 'ig');
```

Comment:
Looking at query suggest, I see the regex does match globally, rather only the first hit. I think we should remove the `g` for consistency,

Reply:
I think the other Facet searches does have the g thought 🤔

Reply:
Yes, they do! That's why I used it here too

Reply:
Isn't that inconsistent with the query suggest though? When I type `t` into the search box, I see only the first `t` is in regular font, while the second `t` is in bold.

If I type `t` into the facet search and one of the results is: "text"

Will the highlighting look like this?
**t**ex**t**

If so, is this desired?

Reply:
it would look like this **t**ex**t**

Reply:
ok, could we check with UX that this is intended?

Reply:
As discussed in scrum, this is intended behavior.